rosidl_typesupport_cpp/resource/action__type_support.cpp.em

```diff
@@ -0,0 +1,68 @@
+// generated from rosidl_typesupport_cpp/resource/action__type_support.cpp.em
+// generated code does not contain a copyright notice
+
+@#######################################################################
+@# EmPy template for generating <action>__type_support.cpp files
+@#
+@# Context:
+@#  - spec (rosidl_parser.ActionSpecification)
+@#    Parsed specification of the .action file
+@#  - subfolder (string)
+@#    The subfolder / subnamespace of the message, usually 'action'
+@#  - type_supports (list of strings)
```

Comment:
@apojomovsky unused `type_supports` list?

Reply:
true that, just removed references to it where unused. thanks!